pkg/monitoring/rules/rules_test.go

```diff
@@ -0,0 +1,37 @@
+package rules
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/testutil"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Rules Validation", func() {
+	var linter *testutil.Linter
+
+	BeforeEach(func() {
+		err := SetupRules("")
+		Expect(err).ToNot(HaveOccurred())
+
+		linter = testutil.New()
+	})
+
+	It("Should validate alerts", func() {
+		linter.AddCustomAlertValidations(testutil.ValidateAlertNameLength, testutil.ValidateAlertRunbookURLAnnotation,
+			testutil.ValidateAlertHealthImpactLabel, testutil.ValidateAlertPartOfAndComponentLabels)
+
+		alerts := ListAlerts()
+		for _, alert := range alerts {
+			problems := linter.LintAlert(&alert)
+			Expect(problems).To(BeEmpty())
+		}
```

Comment:
I think this is possible:

```suggestion
		problems := linter.LintAlerts(alerts)
		Expect(problems).To(BeEmpty())
```

Reply:
I somehow missed it, thanks, updated.